projects/rero/ng-core/src/lib/record/record-statistics/record-statistics.component.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * RERO angular core
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, Input, OnInit } from '@angular/core';
+import { RecordService } from '../record.service';
+import { map } from 'rxjs/operators';
+
+@Component({
+  selector: 'ng-core-record-statistics',
+  templateUrl: './record-statistics.component.html'
+})
+export class RecordStatisticsComponent implements OnInit {
+  // Record PID.
+  @Input()
+  pid: string;
+
+  // Type of resource.
+  @Input()
+  type: string;
+
+  // Record statistics
+  statistics = {};
+
+  /**
+   * Constructor.
+   *
+   * @param _recordService Record service.
+   */
+   constructor(
+    private _recordService: RecordService,
+  ) {}
+
+  ngOnInit(): void {
+    this._getRecordStatistics();
+  }
+
+  /**
+   * Load record statistics.
+   *
+   * @returns Statistics object
+   */
+  private _getRecordStatistics() {
+    return this._recordService.getRecord(this.type, this.pid, 1, {} || null)
+      .pipe(map((record: any) => record.metadata.statistics))
+      .subscribe(results => {
+        this.statistics = results;
+      });
```

Comment:
single line instruction doesn't required "moustaches" --> `.subscribe(results => this.statistics = results)`